build-fail-blacklist

```diff
@@ -456,13 +456,6 @@ recipes/lorma
 recipes/microbecensus
 recipes/minialign
 recipes/nspdk
-recipes/locarna/1.8.7
-recipes/locarna/1.9.1
-recipes/locarna/1.8.11
-recipes/locarna/1.8.12
-recipes/locarna/1.8.10
-recipes/locarna/1.9.0
-recipes/locarna/1.8.9
 recipes/intarna/2.1.0
```

Comment:
What is with this version?

Reply:
Do I misunderstand something here? What do you think is should be changed? 
I removed the old recipes (because I think, they are not needed anymore). Consequentially, I removed the corresponding entries from the blacklist.

Reply:
Dois are added

Reply:
I was asking about `recipes/intarna/2.1.0` should this also be removed? Does it build? I appreciated the cleanup! Thanks for that!

Reply:
thanks, I will check the intarna recipe